frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeTableRow.tsx

```diff
@@ -30,22 +32,37 @@ const CustomServingRuntimeTableRow: React.FC<CustomServingRuntimeTableRowProps>
           id: `draggable-row-${servingRuntimeName}`,
         }}
       />
-      <Td dataLabel="Name">{getServingRuntimeDisplayNameFromTemplate(template)}</Td>
+      <Td dataLabel="Name">
+        <ResourceNameTooltip resource={template.objects[0]}>
+          {getServingRuntimeDisplayNameFromTemplate(template)}
+        </ResourceNameTooltip>
+      </Td>
```

Comment:
Just asked vince if we need to add the `pre-installed` badge here.

Reply:
@lucferbux The issue of labeling pre-installed images just came up in yesterday's UX meeting. We're going to want to add them there, so we should follow the same pattern and label pre-installed runtimes too. So, to answer your question, yes, please add the label. I'm just not sure if pre-installed is the correct term. 

@kaedward What do you think? What term should we use to label runtimes and images that come shipped with RHODS?

Reply:
@vconzola a few suggestions... "Ready-made", "Premade", "Default", "Default runtime", "Red Hat template"?

Reply:
@vconzola And for what it's worth, I think pre-installed gets the idea across as well. It might not technically be "installed", but I think "pre-installed software" or apps are such a common thing that users would understand what that means.

Reply:
OR: "Preconfigured"!

Reply:
@kaedward It's not really a default, and you don't really configure a runtime. But Red Hat template is interesting. Since the PR is already merged, let's leave it as pre-installed for now, and if anyone complains maybe we change it to RH template.